tracer/src/Datadog.Trace.Coverage.collector/Datadog.Trace.Coverage.collector.csproj

```diff
@@ -0,0 +1,18 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.0</TargetFramework>
+        <ImplicitUsings>false</ImplicitUsings>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Microsoft.TestPlatform.ObjectModel" Version="16.4.0" />
```

Comment:
I'm forced to have the dependency, because the collector works by a discovery mechanism through the 
`DataCollector` base class but also uses both `DataCollectorTypeUri` and `DataCollectorFriendlyName` attributes.

https://github.com/DataDog/dd-trace-dotnet/blob/0cdd59936dbf189d7d641a312bea556eb8e38274/tracer/src/Datadog.Trace.Coverage.collector/CoverageCollector.cs#L24-L26